sda-commons-server-s3-testing/README.md

```diff
@@ -58,3 +59,60 @@ public static final S3MockRule S3_MOCK = S3MockRule
      .putObject("bucket", "content.txt", "RUN SDA")
      .build();
 ```
+
+
+## JUnit 5
+
+This module provides the [`S3Extension`](src/main/java/org/sdase/commons/server/s3/testing/S3Extension.java),
+a JUnit 5 extension that is used to automatically bootstrap an AWS S3-compatible object storage instance
+for integration tests.
+
+This is accomplished using [s3mock](https://github.com/findify/s3mock), which
+provides an in memory object storage.
+
+
+To create a S3 Mock instance, register the S3 test extension in your test class:
+
+```
+@RegisterExtension
+static final S3Extension S3_EXTENSION = S3Extension
+      .builder()
+      .build();
+```
+
+The extension takes care to choose a free port for the endpoint. You can access the mock
+URL using `S3_EXTENSION.getEndpoint()`.
+Often one needs to pass the endpoint URL to the constructor of another rule:
+
+```
+@ExtendWith(DropwizardExtensionsSupport.class)
+class DropwizardIT {
+
+  @RegisterExtension
+  static final S3Extension S3_EXTENSION = S3Extension
```

Comment:
Please also add `@Order(0)` here because the right order is not granted otherwise.